docs/index.md

```diff
@@ -4,7 +4,7 @@ has_children: true
 # audience: non-technical researcher folks
 ---
 
-# Cumulus Library
+# [Cumulus Library](https://pypi.org/project/cumulus-library/)
```

Comment:
Hmm, I'm not sure I personally love links like that without context. You cover pip in the separate install docs.

Like, from user perspective: should they click that and go read that instead, or keep reading here? Devs that know what a pypi.org link means, might be able to answer that easily, but not all of our audience.

The ETL has this in its root page:
![image](https://github.com/smart-on-fhir/cumulus-library/assets/1196901/0a24718e-97fe-4606-ab8c-04794bef3b2a)

I was thinking of adding another section below it like:
```
== Source Code
Cumulus ETL is open source. If you'd like to browse its code or contribute changes yourself, the code is on [GitHub](link).
```

Reply:
sure, i'm fine with that